### PR TITLE
Python API: expose stylemap attribute

### PR DESCRIPTION
**Pull request by mandel59**

### CODE REVIEW

**Comment by davelab6** [comment]
@frank-trampe does this look good? Seems fine to me, I propose to merge after 7 days

**Comment by frank-trampe** [comment]
For consistency, it ought to be called os2_stylemap. Otherwise, it looks fine to me.

File: fontforge/python.c [hunk 1]
```diff
@@ -13078,6 +13079,9 @@ static PyGetSetDef PyFF_Font_getset[] = {
     {(char *)"os2_width",
      (getter)PyFF_Font_get_OS2_width, (setter)PyFF_Font_set_OS2_width,
      (char *)"OS/2 width", NULL},
+    {(char *)"stylemap",
```
Comment (davelab6) [inline fontforge/python.c]: Like this, `{(char *)"os_stylemap",` ?

Comment (frank-trampe) [inline fontforge/python.c]: Yes.

**Comment by frank-trampe** [comment]
#2539 replaces this.

**Pull request closed without merging**